android/tools/quest_replay/CMakeLists.txt

```diff
@@ -65,5 +65,6 @@ target_link_libraries(
         android
         log
         OpenXR::openxr_loader)
-
+else()
```

Comment:
I think this might be better conditionally included in android/settings.gradle, if that's possible.

Reply:
Are you talking about updating the settings.gradle like this?

```
if (project.hasProperty("DisableOpenXR")) {
    logger.info('GFXReconstruct Quest Replay app skipped [DisableOpenXR found]')
}
else {
    include ':quest_replay'
    project(':quest_replay').projectDir = file('tools/quest_replay')
}
```

That did not work.  Unless you know how to make it work.